### PR TITLE
Fix exporter.selectivity for idle periods

### PR DESCRIPTION
**Pull request by dominiklohmann**
###  :notebook_with_decorative_cover: Description

Let's not print NaN in JSON.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

n/t

### CODE REVIEW

File: libvast/vast/concept/printable/vast/json.hpp [hunk 1]
```diff
@@ -165,7 +168,7 @@ struct json_printer : printer<json_printer<TreePolicy, Indent, Padding>> {
       while (begin != end) {
         if (!indent())
           return false;
-        if (!(*this)(begin->first))
+        if (!(*this) (begin->first))
```
Comment (dominiklohmann) [inline libvast/vast/concept/printable/vast/json.hpp:171]: clang-format 12 disagrees with CI for these. 🤷‍♂️

File: changelog/2021.04.29-rc2/changes/1514.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The `exporter.hits` metric has been removed.
```
Comment (tobim) [inline changelog/2021.04.29-rc2/changes/1514.md:1]: This is still to be fixed, currently it is still part of the output but always 0. I will open a bugfix PR later (or next week).
You can remove that file from this PR.

File: libvast/vast/concept/printable/vast/json.hpp (hunk 1 above)
Comment (tobim) [reply libvast/vast/concept/printable/vast/json.hpp]: 11 looks better.

Comment (dominiklohmann) [reply libvast/vast/concept/printable/vast/json.hpp]: I agree, hence the last commit.

File: changelog/2021.04.29-rc2/changes/1514.md (hunk 2 above)
Comment (dominiklohmann) [reply changelog/2021.04.29-rc2/changes/1514.md:1]: I'll just fix it within this PR, it's not that hard to remove.

Comment (dominiklohmann) [reply changelog/2021.04.29-rc2/changes/1514.md]: Please check the last commit.

**Review by tobim — APPROVED** [review]

File: changelog/2021.04.29-rc2/changes/1514.md (hunk 2 above)
Comment (tobim) [reply changelog/2021.04.29-rc2/changes/1514.md]: Thanks!

**Pull request merged**